### PR TITLE
start adding FreeBSD support

### PR DESCRIPTION
**Pull request by igalic**
by adding support for executing dmidecode and mapping the `field_names`
to `field-names`

n.b.: this only compiles under a cross-compiler and hasn't been tested
on a real freebsd yet.

### CODE REVIEW

**Comment by igalic** [comment]
doesn't recognize Hetzner yet.

File: src/main.rs [hunk 1]
```diff
@@ -55,6 +59,21 @@ impl RsIdentify {
         self.dmi_values.get(field_name).unwrap()
     }
 
+    #[cfg(target_os = "freebsd")]
+    fn get_dmi_field(&mut self, field_name: &str) -> &Option<String> {
+        let field_name_cmd = field_name.replace('_', "-");
+        if !self.dmi_values.contains_key(field_name) {
+            let output = Command::new("dmidecode")
```
Comment (igalic) [inline src/main.rs:66]: handling a lack of `dmidecode` might be a good idea
unless we make it an explicit dependency on *BSD.

**Review by igalic — COMMENTED** [review]
:woman_shrugging:

File: src/main.rs (hunk 1 above)
Comment (OddBloke) [reply src/main.rs:66]: Given that we can't really do much without it present, I think an explicit dependency is reasonable.

**Pull request merged**